src/ship-gen/shipGenerator.js

```diff
@@ -303,8 +307,8 @@ export class Ship {
             for (let y = newRoom.y1; y <= newRoom.y2; y++) {
                 if (x === newRoom.x1 || x === newRoom.x2 || y === newRoom.y1 || y === newRoom.y2) {
                     if (gameMap.locations[x][y].isTileWalkable()) {
-                        if ((gameMap.locations[x-1][y].isTileAtDepth(RenderOrder.WALL) && gameMap.locations[x+1][y].isTileAtDepth(RenderOrder.WALL))
-                         || (gameMap.locations[x][y-1].isTileAtDepth(RenderOrder.WALL) && gameMap.locations[x][y+1].isTileAtDepth(RenderOrder.WALL))) {
+                        if ((gameMap.locations[x - 1][y].isTileAtDepth(RenderOrder.WALL) && gameMap.locations[x + 1][y].isTileAtDepth(RenderOrder.WALL))
```

Comment:
I don't see any reason these need spacing, unless they're breaking an existing eslint or one is added that addresses this